src/main.js

```diff
@@ -1,5 +1,6 @@
 import Vue from 'vue'
 import App from './App.vue'
+import './axios'
```

Comment:
Here we will import the apiConfig file and it should work` import './api/apiConfig'`

Then you will use the method in the component
```
<script>
import HelloWorld from './components/HelloWorld.vue'
import authService from '@/api/authService';

export default {
  name: 'App',
  components: {
    HelloWorld,
  },
  data() {
    return {
      model: {
        first_name: 'register',
        last_name: 'new last name',
        email: 'mihai.pascu@binarcode.com',
        password: 'secret!',
        password_confirmation: 'secret!',
      }
    };
  },
  async mounted() {
    const result = await authService.register(this.model);
    console.log(result);
  }
}
</script>
```